### PR TITLE
Create sponsor model + tests

### PR DESCRIPTION
**Pull request by Derk-Jan**

### CODE REVIEW

File: spec/models/sponsor_spec.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe Sponsor, type: :model do
+  describe "validations" do
+     it "is invalid without a sponsor" do
+       sponsor = Sponsor.new(sponsor: "")
+       sponsor.valid?
+       expect(sponsor.errors).to have_key(:sponsor)
+      end
+        it "is invalid without a logo" do
```
Comment (arnoFleming) [inline spec/models/sponsor_spec.rb]: please fix your indenting

File: spec/models/sponsor_spec.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe Sponsor, type: :model do
+  describe "validations" do
```
Comment (arnoFleming) [inline spec/models/sponsor_spec.rb]: please add a test for a valid `sponsor`

File: spec/support/devise.rb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 RSpec.configure do |config|
-  config.include Devise::Test::ControllerHelpers, type: :controller
-  config.include Devise::Test::ControllerHelpers, type: :view
-  config.include Warden::Test::Helpers
+#  config.include Devise::Test::ControllerHelpers, type: :controller
```
Comment (arnoFleming) [inline spec/support/devise.rb]: do you need to comment these lines? And then, if yes (but I doubt that), maybe, you could also remove them - we have git for history

File: spec/models/sponsor_spec.rb (hunk 2 above)
Comment (Derk-Jan) [inline spec/models/sponsor_spec.rb]: just noticed that i didn,t saved my changes for the model and tests.  I will do an other push with the changes

Comment (Derk-Jan) [inline spec/models/sponsor_spec.rb]: I added an new valid test. 
Although the model and the tests are a bit arbitrary.  I stated in the model that the presence is required for both attributes sponsor and logo.
Is this true Arno?

File: spec/models/sponsor_spec.rb (hunk 1 above)
Comment (Derk-Jan) [inline spec/models/sponsor_spec.rb]: I fixed this.

File: spec/support/devise.rb (hunk 3 above)
Comment (Derk-Jan) [inline spec/support/devise.rb]: Devise was not yet implemented during the test. We agreed in the team to comment these lines till Devise will be implemented.
I will discuss this tomorrow with my fellow Koala's.

Comment (arnoFleming) [inline spec/support/devise.rb]: If you have no devise, I wonder why the file `spec/support/devise.rb` exists

File: spec/factories/sponsors.rb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
```
Comment (arnoFleming) [inline spec/factories/sponsors.rb]: You're not using this factory. Consider removing this, or using it in your tests

File: spec/models/sponsor_spec.rb [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+      sponsor = Sponsor.new(logo: "")
+      sponsor.valid?
+      expect(sponsor.errors).to have_key(:logo)
+    end
+
+
+    it "is valid with a sponsor and logo" do
+      sponsor = Sponsor.new(sponsor: "Coca Cola", logo: "Logo url")
+      sponsor.valid?
+      expect(sponsor).to be_valid
```
Comment (arnoFleming) [inline spec/models/sponsor_spec.rb]: you call it a bit strange. line 21 is the rspec 2.x way (which is deprecated I believe). 

please rewrite to something like
```ruby
it "is valid with a sponsor and logo" do
  sponsor = Sponsor.new(sponsor: "Coca Cola", logo: "Logo url")
  expect(sponsor.valid?).to eq(true)
end

**Pull request merged**